### DISCUSSION TITLE
Client Assertion signature configuration

### DISCUSSION THREAD

**Discussion by kluangguy1** [keycloak/keycloak#14021] [Q&A]
Dear all,
How should I configure the client assertion signature algo in keycloak v19 if I configure Open ID client authentication as "JWT signed with private key", it seems I cannot locate such option in the UI as below

![v19](https://user-images.githubusercontent.com/108395835/186860335-1c8288f6-31e8-4a6e-ae5c-273b947f7ae6.PNG)

Following is the snippet from v18 UI which have such option
![v18](https://user-images.githubusercontent.com/108395835/186860439-005718ba-0114-4f41-a349-3b909bdee5a3.PNG)

any input is appreciated, thanks!

**Comment by i7a7467** [comment]
Added this configuration to the new security admin console.
keycloak/keycloak-ui#4144
keycloak/keycloak-ui#4145
